### PR TITLE
1.4/1672 bug market add and delete vouchers stab 2

### PR DESCRIPTION
**Pull request by charlesstrange2**
- Some tidying up; mostly cosmetic because linting happened
- pull the code for altering the "online" property voucher that failed because we're now offline into the frontend
- remove code that was just wrong.
https://trello.com/c/Synr3e0M/1672-bug-market-add-and-delete-vouchers

### CODE REVIEW

File: src/pages/Tap.vue [hunk 1]
```diff
@@ -225,6 +231,7 @@ export default {
 
             if (char.match(rxNumber)) {
                 event.preventDefault();
+                1;
```
Comment (AlinChindea) [inline src/pages/Tap.vue]: is this `1` supposed to be here by itself?

Comment (charlesstrange2) [reply src/pages/Tap.vue]: er. no. 😬

**Review by AlinChindea — APPROVED** [review]
🤞 try it!

**Review by AlinChindea — APPROVED** [review]
🍀

**Pull request merged**